CMakeLists.txt

```diff
@@ -39,6 +39,7 @@ include(${CMAKE_SOURCE_DIR}/4build/cmake/CUDA.cmake)
 
 find_package(PythonLibs 2.7 REQUIRED)
 find_package(Numpy REQUIRED)
+find_package(OpenBLAS REQUIRED)
```

Comment:
Yes, absolutely. My bad. Fixed in https://github.com/nebw/cgt/commit/c481b6d052070123d0db78e05192d8a0bdf8472a.